core/templates/dev/head/forms/formBuilder.js

```diff
@@ -598,9 +603,22 @@ oppia.factory('rteHelperService', [
       // Returns a DOM node.
       createRteElement: function(componentDefn, customizationArgsDict) {
         var el = $('<img/>');
-        el.attr('src', componentDefn.iconDataUrl);
+        if (componentDefn.previewUrlTemplate) {
+          el.attr('src',
+            $interpolate(
+              componentDefn.previewUrlTemplate)(
+                angular.extend(customizationArgsDict,
+                  {
+                    explorationId: explorationContextService.getExplorationId()
```

Comment:
Sorry, I can't seem to find an example of an RTE in a non exploration-context page. Are you referring to http://localhost:8181/admin for the admin page?